src/clients/__tests__/KeyknoxClient.test.ts

```diff
@@ -22,63 +29,58 @@ describe('KeyknoxClient', () => {
     jwt = jwtGenerator.generateToken(uuid());
   });
 
-  test('KTC-1', async () => {
-    expect.assertions(8);
-    const value = Buffer.from('value');
-    const meta = Buffer.from('meta');
+  it('KTC-1', async () => {
```

Comment:
As tests can serve as a documentation for the library, we should strive to creating meaningful names for the tests in addition to the short codes that we use internally. It even makes the test a lot easier to read, i.e. you don't have to infer the meaning of the test from its source code